packages/ui-logic/src/weave.spec.ts

```diff
@@ -32,12 +32,16 @@ describe("Weave", () => {
       }
     });
 
+    it("returns error message as is if not JSON", () => {
+      const error = `Can not distribute to 0 recipients`;
+      expect(Weave.tryParseError(error)).toContain(`Can not distribute to 0 recipients`);
```

Comment:
I used them on purpose for consistency's sake. Simon used them in the rest of the `toContain` occurrences and maybe a change in style in a lot of tests is outside the scope of this PR.

Reply:
I think this is a typo, but ok.

Reply:
The original motivation was probably to allow using " and ' unescaped inside of `. But I am not sure if this is really required at the end of the day.

Reply:
I think `Weave.tryParseError('Can not distribute to 0 recipients')` should return `null` since this is not a Weave error. Thus the `Weave` should not be responsible for parsing it.

We can have a wrapper that uses `Weave.tryParseError` and also handles other types of errors.